.github/workflows/sync-automation.yml

```diff
@@ -47,23 +47,19 @@ jobs:
           rm -f .github/dependabot.yml
           git restore --source master -- .github/dependabot.yml
           git add -A && git commit -m "[CI]: Syncing CI changes to ${{ matrix.branch }}"
-          git push origin $prbranch
+          GITHUB_TOKEN=${{ secrets.ORG_GH_TOKEN }} gromit git push . tyk-identity-broker $prbranch --branch $prbranch
```

Comment:
@asutosh would be cleaner to set the GITHUB_TOKEN var as **env** on the step attribute:
```
        env:
          GH_TOKEN: ${{ secrets.ORG_GH_TOKEN }}
```
or even at JOB level